playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -211,12 +212,23 @@ def template_enricher(alert: PrometheusKubernetesAlert, params: TemplateParams):
     )
 
 
+class RegexReplacementStyle(Enum):
+    """
+    Patterns for replacers, either asterisks "****" matching the length of the match, or the string "[REDACTED]"
+    """
+    SameLengthAsterisks = auto()
+    Redacted = auto()
+
+
 class LogEnricherParams(ActionParams):
     """
     :var warn_on_missing_label: Send a warning if the alert doesn't have a pod label
+    :var regex_replacer_pattern: a regex pattern to replace text for example for security reasons
+    :var regex_replacement_style: one of SameLengthAsterisks or Redacted (See RegexReplacementStyle)
     """
-
     warn_on_missing_label: bool = False
+    regex_replacer_pattern: Optional[str] = None
```

Comment:
I think we might need a list of regex matchers
The customer request said they have IPs, and other PIIs and will want to sanitize all
I assume that they probably able to do it using a single regex, but probably easier to do it with a list of matchers

Reply:
My original code had this list of matchers, even with a functionality that replaces them in a cool way like [IP] for ip, [PORT] for port, etc..
However there are edge cases here, that can happened depending on the order of the regex definition. So I removed it.

Reply:
I kept a snippet of the original code, I ran it quickly to show you how it looks like:

for
'Hello 123456 xx12Yo\nGoodbye, 007 agent', when there is a regex to replace a number and a text to replace agent names,
It will print:
Hello [NUMBER] xx[NUMBER]Yo
Goodbye, [AGENT NAME]

Reply:
we can quickly chat about the edge case on Sunday